anta/tests/routing/generic.py

```diff
@@ -98,3 +102,42 @@ def test(self) -> None:
                         if (peer_l3intf := peer_data.get("l3intf")) is not None and peer_l3intf != "":
                             failure_message += f" Interface: {peer_l3intf}."
                         self.result.is_failure(failure_message)
+
+
+class VerifyRoutingTableEntry(AntaTest):
+    """
+    This test verifies that the provided route is present in the routing table of a specified VRF.
```

Comment:
Maybe let's add here taht the test does not check that any other not mentionned route are **not** present.